env_settings.py

```diff
@@ -107,4 +107,55 @@
             },
         }
     },
+
+    'swiss': {
+        'hqusers': {
+            "settings": {
+                "number_of_shards": 2,
```

Comment:
wait no, i don't think it matters since we `update`:
```

In [5]: a = {'settings': { 'replicas': 0 }}

In [6]: a.update({'settings': {}})

In [7]: a
Out[7]: {'settings': {}}
```